images/Makefile

```diff
@@ -35,7 +35,10 @@ ceph.%:
 ceph-toolbox.%: ceph.%
 	@$(MAKE) -C ceph-toolbox PLATFORM=$*
 
-do.build.images.%: ceph.% ceph-toolbox.% ;
+cockroachdb.%:
```

Comment:
if i'm developing locally on one provider I can skip building the other provider, right? For example: `make ceph` or `make cockroachdb`

Reply:
I may currently be possible to build just 1 of the provider images, but either way that would be really nice.

For the `build` target, it looks like it just calls the `images` directory makefile with no specific target: https://github.com/jbw976/rook/blob/cockroachdb/Makefile#L115.  I believe the default target for the images makefile is `build`, which calls `@$(MAKE) cache.images`: https://github.com/jbw976/rook/blob/cockroachdb/images/image.mk#L77

maybe there is a way to pass the provider to build through there?